test/runtime/runtime-python.ts

```diff
@@ -0,0 +1,22 @@
+import { PYTHON_DEFAULT_PRESET,PythonFileGenerator } from "../../";
+import path from "path";
+
+import input from "./generic-input.json";
+
+const generator = new PythonFileGenerator({
+    presets: [PYTHON_DEFAULT_PRESET],
+    
+});
```

Comment:
Remember to generate the models with serialization support: https://modelina.org/docs/languages/python#generate-models-with-json-serializer-and-deserializer-methods